webplugin/js/app/km-rich-text-event-handler.js

```diff
@@ -35,9 +35,10 @@ Kommunicate.attachmentEventHandler= {
         var uploadIconHidden= $applozic(e.target).closest('.km-msg-box-attachment').find('.km-progress-upload-icon').hasClass('n-vis');
         var attachmentDiv= $applozic(e.target).closest('.km-msg-box-attachment').children();
         var msgkey = attachmentDiv[0].dataset.msgkey;
-        var deliveryStatusDiv= $applozic(e.target).closest('.mck-clear').find('.mck-msg-right-muted');      
+        var deliveryStatusDiv= $applozic(e.target).closest('.mck-clear').find('.mck-msg-right-muted');  
+        var fileMetaKey = attachmentDiv[0].dataset.filemetakey;    
         if(Kommunicate.internetStatus) {
-            if(!stopUploadIconHidden && uploadIconHidden) {
+            if((!stopUploadIconHidden && uploadIconHidden) || fileMetaKey == "undefined") {
```

Comment:
@jithinbalu Why undefined in string ?

Reply:
@ab14bhardwaj the value is getting from HTML data attribute that is why undefined is a string